pytest_molecule/__init__.py

```diff
@@ -11,12 +11,95 @@
 from molecule.config import molecule_drivers
 
 
+def pytest_addoption(parser):
+    group = parser.getgroup("molecule")
+    help_msg = (
+        "What marker to add to molecule scenarios when driver is "
+        "unavailable. (ex: skip, xfail)"
+    )
+    default = "skip"
+    dest = "molecule_unavailable_driver"
+
+    group.addoption(
+        "--molecule-unavailable-driver",
+        action="store",
+        dest=dest,
+        default=default,
```

Comment:
maybe this default should be an empty string? otherwise maybe this flag should be required?